app/models/services/turbo-service.ts

```diff
@@ -22,4 +23,20 @@ export default class TurboService extends BaseService<TurboServiceParams> {
   async _run(_logger: Logger): Promise<InvocationResult> {
     return null;
   }
+}
+
+/**
+ * Extends TurboService for testing purposes.
+ */
+export class TestTurboService extends TurboService {
```

Comment:
This should probably be in a file under the 'test' directory.

Reply:
✅